build.gradle

```diff
@@ -45,7 +45,8 @@ ext {
 ext.deps = [
     // Android
     android: 'com.google.android:android:4.1.1.4',
-    
+    supportv4: 'com.android.support:support-v4:23.0.1',
```

Comment:
You need to add `extra-android-m2repository` to the Android dependencies in the `.travis.yml`